hw/dv/sv/common_ifs/index.md

```diff
@@ -1,5 +1,6 @@
-# Common interfaces
-
+---
+title: Common interfaces
```

Comment:
Should we capitalize all words in the title (except for articles, etc)?

Reply:
Thanks, I think for consistency we should capitalize all words in the title.